cmd/configure.go

```diff
@@ -499,6 +503,39 @@ func configureGithub(ctx context.Context, _flags ConfigureGithubFlags) error {
 
 	ctx = events.SetTargetInContext(ctx, workingDir)
 
+	// check if the git repository is a github URI
+	event := shared.CliEvent{}
+	events.EnrichEventWithGitMetadata(ctx, &event)
+
+	// Installing the app is only relevant if we are in a remote linked github repository
+	hasAppAccess := false
+	if event.GitRemoteDefaultOwner != nil && *event.GitRemoteDefaultOwner != "" && event.GitRemoteDefaultRepo != nil && *event.GitRemoteDefaultRepo != "" {
+		hasAppAccess = checkGithubAppAccess(ctx, *event.GitRemoteDefaultOwner, *event.GitRemoteDefaultRepo)
+		if !hasAppAccess {
+			continueAfterInstall := false
+			_, err := charm.NewForm(huh.NewForm(
+				huh.NewGroup(
+					huh.NewSelect[bool]().
+						Title("\n\nSpeakeasy has a github app that can help you setup your repo for SDK generation.\nWould you like to continue after installing the Github App?\n").
+						Options(
+							huh.NewOption("Yes", true),
+							huh.NewOption("No", false),
+						).
+						Value(&continueAfterInstall),
+				),
+			)).ExecuteForm()
+			if err != nil {
+				return err
+			}
+
+			if continueAfterInstall {
+				browser.OpenURL(appInstallURL)
+				logger.Println(styles.Info.Render("Install the Github App then continue with `speakeasy configure github`!\n"))
```

Comment:
Can we not wait until we detect its installed, then continue?

Reply:
We could poll. But I'm not sure that's the best idea.
1. We don't know what kind of caching github has in place
2. Sometimes a user may not have the right permissions to install it by themselves.


Polling just felt kind of weird to me